demisto_sdk/__main__.py

```diff
@@ -782,8 +787,17 @@ def upload(**kwargs):
     DEMISTO_API_KEY environment variable should contain a valid Demisto API Key.
     * Note: Uploading classifiers to Cortex XSOAR is available from version 6.0.0 and up. *
     """
-    if kwargs.pop('zip', False):
-        pack_path = kwargs['input']
+    if kwargs['zip'] or kwargs['input_config_file']:
+        if kwargs.pop('zip', False):
+            pack_path = kwargs['input']
+            kwargs.pop('input_config_file')
+
+        else:
+            config_file_path = kwargs['input_config_file']
+            config_file_to_parse = ConfigFileParser(config_file_path=config_file_path)
+            pack_path = config_file_to_parse.parse_file()
+            kwargs.pop('input_config_file')
+
```

Comment:
```suggestion
    if kwargs['zip'] or kwargs['input_config_file']:
        if kwargs.pop('zip', False):
            pack_path = kwargs['input']
            kwargs.pop('input_config_file')

        else:
            config_file_path = kwargs.pop('input_config_file')
            pack_path = ConfigFileParser(config_file_path=config_file_path).parse_file()
```

Reply:
`PacksZipper` expected to get the  pack_paths as a csv string but it's look like you send it as list 
(it should work even with list because it will parse from csv to a list, but for type matching you should send to `PacksZipper` as CSV string)